jaeger/jaeger_install/rhbjaeger-deploying.adoc

```diff
@@ -1,42 +1,78 @@
 [id="deploying-jaeger"]
 = Deploying Jaeger
-include::modules/ossm-document-attributes.adoc[]
+include::modules/jaeger-document-attributes.adoc[]
 :context: jaeger-deploying
 
 toc::[]
 
-The Jaeger Operator includes a custom resource definition (CRD) file that determines the architecture and configuration settings to use when creating the Jaeger custom resources (CR). You can either install the default configuration or modify the custom resource file to better suit your business requirements.
+The Jaeger Operator includes a custom resource definition (CRD) file that defines the architecture and configuration settings for the Jaeger resources. You can either install the default configuration or modify the file to better suit your business requirements.
 
-Each Jaeger instance is associated with a deployment strategy. You define the strategy is defined in the custom resource file and when you create a Jaeger instance the Operator uses the deployment strategy and configuration options in the CR file to create the objects necessary for the deployment.
-The Jaeger Operator currently supports the following deployment strategies:
+You can deploy Jaeger following different deployment strategies. You define the strategy in the custom resource file. When you create a Jaeger instance the Operator uses this configuration file to create the objects necessary for the deployment.
 
-* *allInOne* (Default) strategy - This strategy is intended for development, testing, and demo purposes. The main backend components, Agent, Collector and Query service, are all packaged into a single executable which is configured (by default) to use in-memory storage.
+.Jaeger custom resource file showing deployment strategy
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: simple-prod
+spec:
+  strategy: production <1>
+----
+
+<1> The Jaeger Operator currently supports the following deployment strategies:
 
+* *allInOne* (Default) strategy - This strategy is intended for development, testing, and demo purposes. The main backend components, Agent, Collector and Query service, are all packaged into a single executable which is configured (by default) to use in-memory storage.
```

Comment:
Since the introductory sentence identifies them all as strategies, do they also need to be labeled strategies in the unordered list? I don't think they do. I think they can just be allInOne, production, and streaming.